sierra_adapter/sierra_merger/src/test/scala/weco/pipeline/sierra_merger/fixtures/RecordMergerFixtures.scala

```diff
@@ -20,18 +20,18 @@ import weco.storage.streaming.Codec._
 
 import scala.concurrent.ExecutionContext.Implicits.global
 
-trait RecordMergerFixtures extends Akka with SQS with SourceVHSFixture {
+trait RecordMergerFixtures extends Pekko with SQS with SourceVHSFixture {
 
-  def withRunningWorker[Record <: AbstractSierraRecord[_], R](
```

Comment:
I renamed the `Record` type parameter to `SierraRecord` here and in a few other places because the compiler was complaining:

> type parameter Record defined in method withRunningWorker shadows class Record defined in package lang. You may want to rename your type parameter, or possibly remove it.